pyscaffold/utils.py

```diff
@@ -209,3 +210,13 @@ def check_setuptools_version():
         raise RuntimeError(
             "ERROR: Your setuptools version is too old (<12).\n"
             "Use `pip install -U setuptools` to upgrade.")
+
+
+def get_git_root():
```

Comment:
Could we move this function into the module ``repo.py`` since it is related to the handling of the git repository. Also please take a look into ``shell.py``, this module provides already a ```git`` command. that means the ``get_git_root`` function could be reduced to something like ``return git('git', 'rev-parse', '--show-toplevel')`` if I got it right.